### ISSUE TITLE
Does `.enable_lora` work on PaliGemma?

### ISSUE THREAD

**Issue by arrufat** [keras-team/keras-nlp#1656]
Hi, I am trying to fine-tune PaliGemma.

The fine-tuning works ok (loss goes down) when I fine-tune the whole model.
However, if I do

```python
pali_gemma_lm.backbone.enable_lora(2)
```

I can see that it has enabled it when printing the model summary (only a few parameters are trainable).
However, with no additional code changes, the loss goes down. Am I missing something?
Thanks in advance.

**Comment by SuryanarayanaY** [comment]
Hi @arrufat ,

IMO, `enable_lora` should work as it is also inheriting Backbone class. Could you please attach the logs on how the loss is actually increasing over epochs?

**Comment by arrufat** [comment]
@SuryanarayanaY My bad, after fiddling a bit more (I changed the optimizer to SGD and increased the learning rate) I got the loss to decrease.
Thank you!
